From 2dd5253b9fa1e242d0c773e4d07be41efea7ac69 Mon Sep 17 00:00:00 2001
From: Leonard Crestez <leonard.crestez@nxp.com>
Date: Mon, 26 Feb 2018 17:51:14 +0200
Subject: [PATCH 0602/5242] MLK-11502 dts: imx7d-sdb: change ramp up delay for
 VDD_SD1 regulator

commit  511537712b638a477948c2b7b64f30281ab97763 from
https://source.codeaurora.org/external/imx/linux-imx.git

Currently the ramp up delay for VDD_SD1 regulator is 1s, the delay
is added in the following patch:

   commit 558c8cde973b9b2abf9a69e3be5a887d38185da3
   MLK-11277 dts: imx7d-sdb: add ramp up delay for VDD_SD1 regulator

Here we find this delay will also impact the sd1 resume time cost, and
1s is too much for the sd1 resume, so we try to reduce the delay time
of this VDD_SD1 regulator, finally 200ms pass our test.

This patch change the ramp up delay for VDD_SD1 regulator from 1s to
200ms.

Signed-off-by: Haibo Chen <haibo.chen@freescale.com>
(cherry picked from commit 16523e1bd8613c38b8182da36fdae99564980e24)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7d-sdb.dts |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/imx7d-sdb.dts b/arch/arm/boot/dts/imx7d-sdb.dts
index 7696a1d..63d0598 100644
--- a/arch/arm/boot/dts/imx7d-sdb.dts
+++ b/arch/arm/boot/dts/imx7d-sdb.dts
@@ -133,7 +133,7 @@
 		regulator-min-microvolt = <3300000>;
 		regulator-max-microvolt = <3300000>;
 		gpio = <&gpio5 2 GPIO_ACTIVE_HIGH>;
-		startup-delay-us = <1000000>;
+		startup-delay-us = <200000>;
 		enable-active-high;
 	};
 
-- 
1.7.9.5

